wsdl/ver10/schema/onvif.xsd

```diff
@@ -7721,6 +7726,11 @@ and sample rate. </xs:documentation>
 					<xs:documentation>Token of a storage configuration.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="StorageStrategy" type="xs:string" minOccurs="0">
```

Comment:
Please move the new element to then end immediately before the xs:any to avoid breaking backward compatibility. As alternative you may defined it as an attribute.